kubejs/server_scripts/unify_outputs.js

```diff
@@ -25,4 +25,13 @@ onEvent('recipes', (event) => {
             }
         });
     });
+
+    materialsToUnify.forEach((material) => {
```

Comment:
Duplicate of the first change I think? I see no reason for this to be here at least.

Reply:
Yes, sorry, I forgot to remove this part 😅